src/SvgUtils.test.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2023 Inclusive Design Research Centre, OCAD University
+ * All rights reserved.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/inclusive-design/adaptive-palette/blob/main/LICENSE
+ */
+"use strict";
+
+import { initAdaptivePaletteGlobals, adaptivePaletteGlobals } from "./GlobalData";
+import { bciToBlissaryId, bciAvIdToString } from "./SvgUtils";
+
+describe("SvgUtils module", () => {
+
+  // The `singleBciAvId` is taken from the BMW json for "CONJ." The
+  // `bciAvIdArray` is also from the BMW json file using the codes for
+  // "VERB+EN".  The `expectedX` constants are based on a manual lookup of the
+  // blissary ids.
+  const singleBciAvId = 23409;                // CONJ.
```

Comment:
A more structured way to organize test cases from line 22-26 could be:

```
const testCases = {
  "numberInput": {
    "input": 23409,
    "expected": "B823"
  },
  "arrayInput": {
    "input": [ 12335, "/", 8499 ],
    "expected": "B106/B12"
  },
  "nonexistentBciAvId": {
    "input": 1,
    "expected": undefined
  }
  ...
}
```

The test then loop through `testCases` with the same code to handle each input and check the expected.

Reply:
I tried the loop, but have not committed the code.  I found it more difficult to see what is being tested.  I prefer having readable explicit tests to having to think about the parameters in a loop where the tests are executed, and then look up their values in a different block of code.  The loop technique feels opaque to me.

Reply:
Understand. Please go ahead with the approach that works best for you.